### DISCUSSION TITLE
Can i change damage type on weapon using active effect?

### DISCUSSION THREAD

**Discussion by chatchayanan** [League-of-Fabulous-Developers/FoundryVTT-Fabula-Ultima#118] [Q&A] [answered]
I just tried using this game system and wondered if there's a way to change the damage type of the weapon that player character is wielding using active effects. I've tried writing it myself. but it wasn't successful at all.

**Comment by spyrella** [comment]
Sorry for late reply,
If you're interested I did write a macro for easily changing damage/range type on the fly as well as temporarily adding modifiers and rolling it.

You can check out the code for the macro over at: https://github.com/spyrella/pfu-macros/blob/main/weapon-customizer.js

**Comment by spyrella** [comment, marked as answer]
To follow up, in the new version you can also use the inline command `@WEAPON[dark]` for changing a weapons type.
